privacy.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: privacy
+title: Privacy Notice
+---
+
+# Foreman Privacy Policy
+
+This document describes the data Foreman collects about you, why it is
+collected, what is done with it, and your options for how it's processed.
+
+The Foreman project does not share data with 3rd parties, except for hosting
+requirements (e.g. GitHub, YouTube) as detailed below, or when required to by
+law.
+
+# Types of data we collect
+
+* Contact details
+  * Name
+  * Email address
+* Data that identifies you
+  * IP addresses
+  * Browser UserAgent strings
+  * Debug archives
+  * Answers to survey questions
+
+# How and why we use your data
+
+## The website
+
+The Foreman website is a static site which does not track, collect, store
+information or set cookies. There is no account for the website itself. The
+webserver (Apache) logs will contain IP addresses, timestamps, and UserAgent
+strings - these are log-rotated and deleted after 4 weeks.
+
+The webserver logs are primarily used to produce aggregated statistics about
+the community, such as popularity of different operating system packages,
+plugins, etc.
+
+* Legal basis for this data usage: Legitimate interest
+
+The website does embed YouTube videos, see [YouTube's privacy policy][YouTube]
+for details. These videos do not autoplay.
+
+## Codebase
+
+The Foreman community use [GitHub](https://github.com) to manage the code that
+comprises the Foreman project. Refer to [GitHub's privacy policy][GitHub] for
+details on how GitHub handles the data you posts there. We also use
+non-identifiable data from the codebase to provide aggregated statistics such
+as average time to merge, number of open pull requests, etc.
+
+* Legal basis for this data usage: Legitimate interest
+
+The Foreman community also run a bug tracker using the
+[Redmine](http://www.redmine.org) software. When you sign up for a Redmine
+account, you provide us with an email address and a name. We only use this
+email within Redmine to contact you about bugs you have reported or subscribed
```

Comment:
This email address is also used to link to git authors. It can be linked manual as well. This allows us to let you close an issue from a git commit with the correct attribution. We also build a more complete user activity because of this.

Reply:
Thanks, I've restructed the Redmine section for this.